data/categories.txt

```diff
@@ -1249,7 +1249,7 @@ amenity-place_of_worship
 en:place of worship|4temple|sights|U+1F64F|U+26EA
 ru:3храм|церковь|достопримечательность
 uk:3храм|церква|собор|пам’ятка|алтарь|мавзолей|пам’ятні місця
-de:Anbetungsstätte|Sehenswürdigkeit
+de:Anbetungsstätte|Kirche|Dom|Tempel|Kathedrale|Kapelle|Moschee|Sehenswürdigkeit
```

Comment:
Ok, but Chinese and Japanese seem to be quite verbose, too. (I don't speak those languages, though.)